marlowe-playground-client/test/BridgeTests.purs

```diff
@@ -47,11 +47,11 @@ serializationTest =
 
       alicePk = PK "deadbeef"
 
-      aliceAcc = AccountId (fromIntegral 0) alicePk
+      aliceAcc = AccountId (fromInt 0) alicePk
```

Comment:
oh, isn't there an Integral instance?!

Reply:
No. I didn't think it was a good idea. It lulls you into the false belief that you can freely convert between `Int` and `BigInteger`.

Reply:
not convinced, doesn't `fromInt` do exactly the same?

Reply:
No, it's one-way. It's `BigInteger -> Int` that troubles me. That should be `BigInteger -> Maybe Int`.

Reply:
fair enough